release-helm.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex
+
+git clone -b dev-almerico-AAE-2501-helm https://${GITHUB_TOKEN}:x-oauth-basic@github.com/Activiti/activiti-cloud-application.git
```

Comment:
we should change this branch name to develop

Reply:
done